### PR TITLE
Fix https://github.com/nrc/rustfmt/issues/278

### PR DESCRIPTION
**Pull request by sinhpham**

### CODE REVIEW

File: tests/source/struct_lits.rs [hunk 1]
```diff
@@ -68,3 +68,11 @@ fn issue201() {
...
     let s = S{a: S2{    .. c}, ..  b};
 }
+
+fn issue278() {
+    let s = S {
+        a: 0,
+        //       
+        b: 0,
+    };
+}
```
Comment (nrc) [inline tests/source/struct_lits.rs]: Could you add a test with the empty comment in the middle, e.g., 
```
// foo
// 
// bar
```

**Comment by nrc** [comment]
r+ with the extra test

File: tests/source/struct_lits.rs (hunk 1 above)
Comment (sinhpham) [inline tests/source/struct_lits.rs]: Added.

**Comment by nrc** [comment]
Thanks!
